### ISSUE TITLE
About the performance of 1/8 labels.

### ISSUE THREAD

**Issue by yh-pengtu** [charlesCXK/TorchSemiSeg#14]
I follow the default setting of codes and obtained 71.687% mIoU when training with 1/8 labels on pascal voc，but this paper reported 73.67% mIoU. Why the performance gap so huge, and anyone can tell me how to improve this code? Thanks very much!

**Comment by charlesCXK** [comment]
Hi, could you provide more training details? For example, running environment, batch size, etc.

**Comment by frank-xwang** [comment]
Hi @charlesCXK, thank you for releasing the code! I obtained similar results (mean_IU 71.590% mean_IU_no_back 70.518% mean_pixel_ACC  93.678%) by using the following script: TorchSemiSeg/exp.voc/voc8.res50v3+.CPS/script.sh. I used 4 RTX 2080Ti GPUs, pytorch 1.7 and cuda10.1.

**Comment by frank-xwang** [comment]
Hi @charlesCXK, I tried the one with CutMix augmentation, it is possible to reproduce the performance reported in the paper, i.e. mean_IU 73.907% mean_IU_no_back 72.927% mean_pixel_ACC  94.073%. 

In summary, I can reproduce results of the setting with CutMix and can not reproduce results of the setting without CutMix. Once again, thanks for releasing the code!

**Comment by charlesCXK** [comment]
Hi, @frank-xwang 
It is so strange. I trained the released code (voc8.res50v3+.CPS) in this repo with CPS weight = 1.5 and 2, both the results are 73.2+. I find your environment is different from mine and I am not sure whether this leads to the strange result. By the way, could you reproduce CPS w/o cutmix on other splits, i.e., 1/4 or 1/2?

**Comment by frank-xwang** [comment]
Sure. I can try other settings.

**Comment by charlesCXK** [comment]
We have released the [training log](https://pkueducn-my.sharepoint.com/:t:/g/personal/pkucxk_pku_edu_cn/ERl1pm99zhFIvIIB8y82WiIB13AQ9Hd8FyrJQ5v3fpP0vg?e=Nd0f1M) and [pretrained model](https://pkueducn-my.sharepoint.com/:u:/g/personal/pkucxk_pku_edu_cn/ESx6vF1dapJGkbsLWHnHSakBdFOkooQcIFeDpCRTVJS8Iw?e=zMCvhj) for this experiment (voc8.res50v3+.CPS) on OneDrive. Hope this will be helpful!

**Comment by frank-xwang** [comment]
Thanks! I will take a look.

**Comment by PLoic** [comment]
Hi @charlesCXK  thanks for this work, could you also release the training log for the cityscape experiment ? 

Thanks

**Comment by charlesCXK** [comment]
@PLoic 
Please check the [training log](https://pkueducn-my.sharepoint.com/:t:/g/personal/pkucxk_pku_edu_cn/EdRRCsS2KtFGoTophitkLh0BnA40ZPBmuVKhWEV-biF2lw?e=8LaE88) of `city8.res50v3+.CPS` experiment.

**Comment by PLoic** [comment]
@charlesCXK  Thanks a lot for your quick respond !
